src/profile-logic/process-profile.js

```diff
@@ -1750,13 +1785,25 @@ export function serializeProfile(profile: Profile): string {
  */
 function _unserializeProfile({
```

Comment:
Ok, so as long as the code in this function handles both the old and the new format, we don't need to add an upgrading step *prior to unserialization*. That seems ok for now but it won't scale forever.